crates/debugger/Cargo.toml

```diff
@@ -0,0 +1,31 @@
+[package]
+name = "wasmtime-internal-debugger"
+authors.workspace = true
+version.workspace = true
+description = "INTERNAL: Wasmtime's guest-debugger functionality"
+license = "Apache-2.0 WITH LLVM-exception"
+documentation = "https://docs.rs/wasmtime-debugger"
+repository = "https://github.com/bytecodealliance/wasmtime"
+categories = ["no-std"]
+readme = "README.md"
+keywords = ["debugger"]
+edition.workspace = true
+rust-version.workspace = true
+
+[lints]
+workspace = true
+
+[dependencies]
+wasmtime = { workspace = true, features = ["debug", "std", "async"] }
+tokio = { workspace = true, features = ["rt", "sync", "macros"] }
+anyhow = { workspace = true }
+log = { workspace = true }
+
+[dev-dependencies]
+# Depend on `wasmtime` again to get `cranelift` so we can write unit
+# tests that build modules.
+wasmtime = { workspace = true, features = ["debug", "std", "async", "cranelift", "wat"] }
```

Comment:
You can drop debug/std/async here since that'll come in through the normal dependency